x/datadeal/abci.go

```diff
@@ -36,18 +36,19 @@ func EndBlocker(ctx sdk.Context, keeper keeper.Keeper) {
 
 		if tallyResult.IsPassed() {
 			dataSale.Status = types.DATA_SALE_STATUS_DELIVERY_VOTING_PERIOD
-			if dataSale.VerifiableCid != string(tallyResult.ConsensusValue) {
-				panic("invalid verifiable CID consensus value")
+			if dataSale.DataHash != string(tallyResult.ConsensusValue) {
+				panic("invalid dataHash consensus value")
```

Comment:
Thanks, I also think panic(err) is not needed. So, I remove it.
But, I wonder if there is no need to change the status `DATA_SALE_STATUS_VERIFICATION_FAILED` in this case.

Reply:
It might be already passed if I understand correctly. So the status must be `DATA_SALE_STATUS_DELIVERY_VOTING_PERIOD`.

In verification vote, there is no meaning in consensus value. The votes only determine if the data is valid or not :)

Reply:
Yes. I think so too.

Reply:
Okay, I see. Thank you!